packages/vaadin-combo-box/src/vaadin-combo-box-dropdown.js

```diff
@@ -88,7 +112,8 @@ class ComboBoxDropdownElement extends DisableUpgradeMixin(mixinBehaviors(IronRes
       <vaadin-combo-box-overlay
         id="overlay"
         hidden$="[[hidden]]"
-        opened="[[opened]]"
+        position-target="[[positionTarget]]"
+        opened="{{opened}}"
```

Comment:
https://github.com/vaadin/web-components/pull/2497 uses a two-way binding for a different property `_overlayOpened`. That doesn't exist here, but I couldn't observe any negative effects from using a two-way binding on `opened` itself.